agent/app/agent_capability.go

```diff
@@ -498,6 +503,24 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendGuardDutyCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	isEnabled, err := pathExists(guardDutyServiceFilePath, false)
+	if err != nil {
+		logger.Warn(fmt.Sprintf("Skipping enabling GuardDutyCapabilities. Error: %v", err))
```

Comment:
nit. Can we use this format for log message?

```
logger.Warn("GuardDuty Capability: Cannot find GuardDuty service file path. GuardDuty capabilities will not be enabled on this container instance",   logger.Fields{
    field.Error:  err,
})
```

Reply:
Updated to use this format